cmip6_downscaling/data/cmip.py

```diff
@@ -163,38 +163,38 @@ def cmip():
     return model_dict, data
 
 
-def load_cmip_dictionary(
-    activity_ids=["CMIP"],
-    experiment_ids=["historical"],  # , "ssp126", "ssp245",  "ssp585"
-    member_ids=["r1i1p1f1"],
-    source_ids=["MIROC6"],  # BCC-CSM2-MR"]
-    table_ids=["day"],
-    grid_labels=["gn"],
-    variable_ids=["tasmax"],
-    return_type='zarr',
-):
-    """Loads CMIP6 GCM dataset dictionary based on input criteria.
+def load_cmip(
+    activity_ids: list = ["CMIP"],
+    experiment_ids: list = ["historical"],
+    member_ids: list = ["r1i1p1f1"],
+    source_ids: list = ["MIROC6"],
+    table_ids: list = ["day"],
+    grid_labels: list = ["gn"],
+    variable_ids: list = ["tasmax"],
+    return_type: list = 'zarr',
+) -> xr.Dataset:
+    """Loads CMIP6 GCM dataset based on input criteria.
 
     Parameters
     ----------
     activity_ids : list, optional
-        [activity_ids in CMIP6 catalog], by default ["CMIP", "ScenarioMIP"],
+        activity_ids in CMIP6 catalog, by default ["CMIP", "ScenarioMIP"],
     experiment_ids : list, optional
-        [experiment_ids in CMIP6 catalog], by default ["historical", "ssp370"],  ex:#  "ssp126", "ssp245",  "ssp585"
+        experiment_ids in CMIP6 catalog, by default ["historical", "ssp370"],  ex:#  "ssp126", "ssp245",  "ssp585"
     member_ids : list, optional
-        [member_ids in CMIP6 catalog], by default ["r1i1p1f1"]
+        member_ids in CMIP6 catalog, by default ["r1i1p1f1"]
     source_ids : list, optional
-        [source_ids in CMIP6 catalog], by default ["MIROC6"]
+        source_ids in CMIP6 catalog, by default ["MIROC6"]
     table_ids : list, optional
-        [table_ids in CMIP6 catalog], by default ["day"]
+        table_ids in CMIP6 catalog, by default ["day"]
     grid_labels : list, optional
-        [grid_labels in CMIP6 catalog], by default ["gn"]
+        grid_labels in CMIP6 catalog, by default ["gn"]
     variable_ids : list, optional
-        [variable_ids in CMIP6 catalog], by default ['tasmax']
+        variable_ids in CMIP6 catalog, by default ['tasmax']
 
     Returns
     -------
-    [dictionary]
+    ds : xr.Dataset
```

Comment:
This function also returns a zarr group, depending on the value of `return_type`